automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/sam/SamApiSpec.scala

```diff
@@ -0,0 +1,68 @@
+package org.broadinstitute.dsde.firecloud.test.api.sam
+
+import org.broadinstitute.dsde.firecloud.api.Sam
+import org.broadinstitute.dsde.firecloud.api.Sam.user.UserStatusDetails
+import org.broadinstitute.dsde.firecloud.auth.{AuthToken, ServiceAccountAuthToken}
+import org.broadinstitute.dsde.firecloud.config.{Config, Credentials, UserPool}
+import org.broadinstitute.dsde.workbench.model.{WorkbenchUserServiceAccount, WorkbenchUserServiceAccountName}
+import org.broadinstitute.dsde.firecloud.dao.Google.googleIamDAO
+import org.broadinstitute.dsde.workbench.google.model.GoogleProject
+import org.scalatest.{FreeSpec, Matchers}
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class SamApiSpec extends FreeSpec with Matchers {
+  val anyUser: Credentials = UserPool.chooseAnyUser
+  val userAuthToken: AuthToken = anyUser.makeAuthToken()
+
+  def petName(userInfo: UserStatusDetails) = WorkbenchUserServiceAccountName(s"pet-${userInfo.userSubjectId}")
+
+  def removePet(userInfo: UserStatusDetails): Unit = {
+    Sam.admin.deletePetServiceAccount(userInfo.userSubjectId)(UserPool.chooseAdmin.makeAuthToken())
+    val remove = googleIamDAO.removeServiceAccount(GoogleProject(Config.Projects.default), petName(userInfo))
```

Comment:
I think Sam delete pet should remove it from Google:
https://github.com/broadinstitute/sam/blob/develop/src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala#L185

Did you find that not to be the case?

Reply:
Huh.  Yes I did find it necessary to explicitly remove the SA from Google.

Reply:
I'm having success without it now, so I'm going to take it out.

Reply:
ok cool

Reply:
(nope, I take that back)

Reply:
https://broadinstitute.atlassian.net/browse/GAWB-2867

Reply:
thank you